src/test/java/com/exasol/containers/wait/strategy/AbstractServiceWaitStrategyTest.java

```diff
@@ -1,15 +1,14 @@
 package com.exasol.containers.wait.strategy;
 
-import static com.exasol.containers.ExasolContainerConstants.EXASOL_CORE_DAEMON_LOGS_PATH;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTimeout;
-import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.when;
 
 import java.io.IOException;
 import java.time.Duration;
+import java.time.Instant;
 
-import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
```

Comment:
I see this import, but I don't see the Tag.

Reply:
I added the tag to a single test:

```java
    @Tag("slow")
    @Test
    void testWaitUntilReadyTimesOut() throws IOException, InterruptedException {
```